src/app/teams/teams-view.component.ts

```diff
@@ -277,10 +277,20 @@ export class TeamsViewComponent implements OnInit, OnDestroy {
   }
 
   addMembers(selected: any[]) {
-    this.dialogsLoadingService.start();
     const newMembershipDocs = selected.map(
       user => this.teamsService.membershipProps(this.team, { userId: user._id, userPlanetCode: user.planetCode }, 'membership')
     );
+    const checkActiveRequest = this.requests.some((req: any) => {
+      if (req.userId === newMembershipDocs[0].userId) {
+        this.changeMembership('added', req);
```

Comment:
In general, you should avoid side effects in array functions.  This PR needs a lot of changes so no need to fix this exact problem, but wanted to note this for future reference.